src/documentation/0019-package-json/index.md

```diff
@@ -130,7 +130,7 @@ All those properties are used by either `npm` or other tools that we can use.
 
 This section describes the properties you can use in detail. We refer to "package" but the same thing applies to local applications which you do not use as packages.
 
-Most of those properties are only used on the <https://www.npmjs.com/>, other by scripts that interact with your code, like `npm` or others.
+Most of those properties are only used on the <https://www.npmjs.com/>, others by scripts that interact with your code, like `npm` or others.
```

Comment:
Not your change, but since we're already touching this line, can we get rid of the extra `the`?

```suggestion
Most of those properties are only used on <https://www.npmjs.com/>, others by scripts that interact with your code, like `npm` or others.
```

Reply:
@jgb-solutions I suggest you merge the suggestion from @sagirk  so I can merge this PR. 👍